MMOCoreORB/src/server/zone/objects/creature/commands/RequestSetStatMigrationDataCommand.h

```diff
@@ -51,6 +51,31 @@ class RequestSetStatMigrationDataCommand : public QueueCommand {
 			return GENERALERROR;
 		}
 
+		if (creature->isInCombat()) {
+			creature->sendSystemMessage("You cannot migrate stats in combat!");
+			return 0;
+		}
+
+		//BuildingObject* buildingObj = cast<BuildingObject*>( creature->getParentRecursively(SceneObject::SALONBUILDING));
+		ManagedReference<SceneObject*> obj = creature->getParentRecursively(SceneObjectType::SALONBUILDING);
+
+		if (obj == nullptr) {
+			creature->sendSystemMessage("You must be inside an Image Design tent in order to perform that action.");
```

Comment:
This is sufficient, albeit a bit vague. Any opposition to `"You must be inside an Image Design tent in order to migrate your stats."`?